boards/posix/native_posix/doc/board.rst

```diff
@@ -162,8 +162,12 @@ Application tests using the ``ztest`` framework will exit after all
 tests have completed.
 
 If you want your application to gracefully finish when it reaches some point,
-you may add a conditionally compiled (CONFIG_BOARD_NATIVE_POSIX) call to
-``main_clean_up(exit_code)`` at that point.
+you may add a conditionally compiled (CONFIG_ARCH_POSIX_) call to
+``posix_exit(int status)`` at that point.
+
+.. _CONFIG_ARCH_POSIX:
+   ../../../../reference/kconfig/CONFIG_ARCH_POSIX.html
```

Comment:
do not use file reference likes this, you refer to the option using

 :option:`CONFIG_ARCH_POSIX`

Reply:
Thanks, fixed. The other option in the peripherals I'm fixing in the shell PR to avoid merge conflicts